canonical/snapcraft.io
# Wd 856 refactor try except

## Done
- Register `StoreAPI`, `API` and `PublisherAPI` error handlers
- Remove all try... except blocks whose exceptions have been handled in the error handlers
## How to QA
-  Visit https://snapcraft-io-4180.demos.haus/
- This PR does not add or update any feature, neither does it fix a bug, it is a chore that can simply be validated by ensuring all routes still works as expected
- Also, by triggering an error in the application and checking to ensure the app does not crash but rather catches and handles the error(s) appropriately
## Issue / Card
Fixes #3230 

## Screenshots
